test/serialisers/json-0.6.js

```diff
@@ -285,6 +285,40 @@ describe('JSON 0.6 Serialiser', function() {
       });
     });
 
+    it('serialises enum with content', function () {
+      var enumeration = new minim.Element();
+      enumeration.element = 'enum';
+      enumeration.attributes.set('enumerations', []);
+      enumeration.content = [new minim.elements.String(null)];
```

Comment:
The enumerations content (value) shouldn't be an array per the spec. It should be an element itself.

<img width="891" alt="messages image 2872221322" src="https://user-images.githubusercontent.com/44164/37556881-7b0cc36a-29b9-11e8-9945-5b01a5ff704d.png">